.gitignore

```diff
@@ -33,6 +33,7 @@ pip-wheel-metadata
 # Documentation
 */_build/*
 */site/*
+site/
```

Comment:
Possibly you meant this:

```suggestion
/site
```

Isn't this covered by line 35?

Reply:
I was surprised too, but no, line 35 does not ignore the root `site` directory :confused: